library/management/chat/src/testmod/resources/quilt.mod.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
oh heck this might have been the most thorough testmod QMJ that i've ever seen; unfortunately i don't know if this means whenever we should cut the tall poppy or leave it alone :P

Reply:
Im pretty sure this is just copied+modified from the client commands one? 😅

Reply:
wait heck, it does that; oh dear, we'll make 'em consistent later :P